NitroxClient/GameLogic/Bases/Spawning/Furniture/FurnitureSpawnProcessor.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using NitroxModel.Logger;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Bases.Spawning.Furniture
+{
+    public abstract class FurnitureSpawnProcessor
+    {
+        private static readonly Dictionary<TechType, FurnitureSpawnProcessor> processorsByType = new Dictionary<TechType, FurnitureSpawnProcessor>();
+
+        protected abstract TechType[] ApplicableTechTypes { get; }
```

Comment:
Problem is that `NitroxClient/MonoBehaviours/ThrottledBuilder` and `NitroxClient/GameLogic/Building` are looking for a `ConstructableBase` component and then executing `BasePieceSpawnProcessor`. Furniture just have a `Constructable` component and therefor have to be handled differently. I could integrate them but imo this will cause more confusion.